staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -544,6 +544,14 @@ func (c *Cacher) Watch(ctx context.Context, key string, opts storage.ListOptions
 	} else if selectorNamespace, ok := pred.Field.RequiresExactMatch("metadata.namespace"); ok {
 		scope.namespace = selectorNamespace
 	}
+
+	var coreNamespace = schema.GroupResource{Group: "", Resource: "namespaces"}
```

Comment:
nit: move this to a private package variable outside the function so we don't have to redefine it on every invocation